File: SADC-App/src/app/models/Planting.ts
import { Farm } from './Farm';
import { Field } from './Field';
import { Seed } from './Seed';

export class Planting {
id: number;
plantingDate: Date;
harvest: string;
rainAmount: number;
typePlanting: string;
weatherPlanting: string;
airMoisture: number;
seedId: number;
seed: Seed;
seedAmount: number;
fertilizing: string;
farmId: number;
farm: Farm;
fieldId: number[];
fields: Field[];
}

File: SADC.Domain/Planting.cs
using SADC.Domain.Enums;

namespace SADC.Domain
{
    public class Planting
    { 
        public int Id { get; set; }
        public DateTime PlantingDate { get; set; }
        public string Harvest { get; set; }
        public decimal RainAmount { get; set; }
        public TypePlanting TypePlanting { get; set; }
        public string WeatherPlanting{ get; set; }
        public decimal AirMoisture { get; set; }
        public int SeedId { get; set; }
        public Seed Seed { get; set; }
        public decimal SeedAmount { get; set; }
        public string Fertilizing { get; set; }
        public int FarmId { get; set; }
        public Farm Farm { get; set; }
        public IEnumerable<PlantingField> Fields { get; set; }

    }
}
